src/main/java/frc/robot/Robot.java

```diff
@@ -19,23 +27,38 @@
     final private RobotMap robotMap = new PracticeBot();
     final private ExampleSubsystem exampleSubsystem = new ExampleSubsystem(robotMap);
 
+    public static XboxController driveController = new XboxController(1);
+
     private Command autonomousCommand;
     final private SendableChooser<Command> chooser = new SendableChooser<>();
 
+    UsbCamera camera1;
+    UsbCamera camera2;
+    VideoSink videoSink;
+    boolean activeCamera = false;
+    NetworkTableEntry xEntry;
+
     /**
      * This function is run when the robot is first started up and should be used
      * for any initialization code.
      */
     @Override
     public void robotInit() {
         // Initialize OI here
-
+        camera1 = CameraServer.getInstance().startAutomaticCapture(0);
+        camera2 = CameraServer.getInstance().startAutomaticCapture(1);
+        videoSink = CameraServer.getInstance().getServer();
+        NetworkTableInstance inst = NetworkTableInstance.getDefault();
+        NetworkTable table = inst.getTable("Table Thingy");
```

Comment:
What does this table do?

Reply:
Fancy stuffs for the cool vision code sean is writing

Reply:
Actually I really dont know please explain